src/explore-education-statistics-common/src/modules/charts/util/domainTicks.ts

```diff
@@ -177,15 +177,13 @@ export function getMinorAxisDomainTicks(
 }
 
 export function getMajorAxisDomainTicks(
-  chartData: ChartData[],
+  majorAxisCategories: string[],
```

Comment:
It was because I need to know the length of `majorAxisCategories` for the tick width calculation, so this was just to avoid doing `chartData.map(({ name }) => name)` twice.